app/scripts/controllers/dashboard.coffee

```diff
@@ -544,4 +544,22 @@ angular.module('edudashAppCtrl').controller 'DashboardCtrl', [
         $scope.anchorScroll = () ->
             $anchorScroll()
 
+        $scope.showModal = (total) ->
```

Comment:
schoollistmodal is the modal itself, so the open modal could not be there, we could move it to schoollist if you prefer